source/module_basis/module_nao/projgen.cpp

```diff
@@ -23,7 +23,7 @@ void projgen(const int l,
     assert(rcut < r[nr - 1]);
     assert(std::is_sorted(r, r + nr));
 
-    std::vector<double> dr(nr - 1);
+    std::vector<double> dr(nr);
```

Comment:
why is this 'nr'?

Reply:
> why is this 'nr'?

`std::adjacent_difference` returns with the same size as input.